src/woo_search/search_index/tasks.py

```diff
@@ -160,6 +160,7 @@ def index_document(
     omschrijving: str,
     creatiedatum: date,
     registratiedatum: datetime,
+    gepubliceerd_op: datetime,
```

Comment:
```suggestion
    gepubliceerd_op: datetime | None,
```
for backwards compatibiity